src/NzbDrone.Core/RemotePathMappings/RemotePathMappingRepository.cs

```diff
@@ -16,6 +16,13 @@ public RemotePathMappingRepository(IMainDatabase database, IEventAggregator even
         {
         }
 
+        public new void Delete(int id)
+        {
+            var model = Get(id);
+            base.Delete(id);
+            ModelDeleted(model);
+        }
```

Comment:
This is needed so that we get updates if a remote path mapping is removed.  `BasicRepository` doesn't publish events on deletion.